lib/decorators/cron.decorator.ts

```diff
@@ -29,6 +29,11 @@ export interface CronOptions {
    * If you have code that keeps the event loop running and want to stop the node process when that finishes regardless of the state of your cronjob, you can do so making use of this parameter. This is off by default and cron will run as if it needs to control the event loop. For more information take a look at [timers#timers_timeout_unref](https://nodejs.org/api/timers.html#timers_timeout_unref) from the NodeJS docs.
    */
   unrefTimeout?: boolean;
+
+  /**
+   * This flag indicates whether the job will be executed at all (true by default).
```

Comment:
```suggestion
   * This flag indicates whether the job will be executed at all.
   * @defaults true
```

Reply:
actually, the default is `false`, right? :thinking:

Reply:
My bad 😄